nupic/algorithms/anomaly.py

```diff
@@ -140,6 +148,17 @@ def compute(self, activeColumns, predictedColumns,
     if self._movingAverage is not None:
       score = self._movingAverage.next(score)
 
+    # apply binary discretization if required
+    if self._binaryThreshold is not None:
```

Comment:
Are you adding this for a real need that you have or just because you think it may be useful to someone? In the former case, which is useful for you?